DEPS

```diff
@@ -471,6 +471,9 @@ deps = {
   'src/third_party/dart/third_party/pkg/string_scanner':
    Var('dart_git') + '/string_scanner.git@da9142cf9809e7e1364144b8193ec60d87f0a4b8',
 
+  'src/third_party/dart/third_party/pkg/tar':
```

Comment:
This needs to be added outside of automatically-maintained list of dependencies. Otherwise it will be automatically removed by next dart->engine roller.

Reply:
Even when the package is part of the dart sdk DEPS?

See https://github.com/dart-lang/sdk/blob/main/DEPS#L182

Reply:
It won't be removed by the roller, now that it is in the SDK DEPS. The previous rolls rolled versions from before it landed.